packages/internal/bridge/sdk/src/lib/validation.ts

```diff
@@ -0,0 +1,151 @@
+import { ethers } from 'ethers';
+import { BridgeBundledTxRequest, FungibleToken } from 'types';
+import { NATIVE } from 'constants/bridges';
+import { BridgeConfiguration } from '../config';
+import { BridgeError, BridgeErrorType, withBridgeError } from '../errors';
+import { isChildETH, isRootIMX } from './utils';
+
+// TODO consider moving these to be member methods of the config class
```

Comment:
Since these all perform operations mostly around the BridgeConfiguration object, it might make sense that these become part of that class. Can follow up on this in a future PR if we believe so.